### PR TITLE
Adding processor for graphic inputs

### LINKED ISSUE #82: Wrong geometry representation in GraphicInput
The graphic input relies on package `com.vividsolutions.jts.geom` whereas the rest uses `org.locationtech.jts.io.geojson`. Search for occurences of `com.vividsolutions.jts.geom` throughout the project.

### PR DESCRIPTION
**Pull request by ckittl**
Partly solves #73 (introduces an `GraphicInputProcessor` for all `GraphicInput` entities) and is based on #74.

when this one is merged, we still need processors for remaining `InputEntity`s (`OperatorInput`, `AssetCharacteristicInput`-subclasses, `RandomLoadParameter` and `AssetTypeInput` - _when addressing this, it might be possible to extend the `AssetInputProcessor` to a full `InputEntityProcessor` instead of splitting up logic to new processors_)

Additionally this pull request closes #82.

### CODE REVIEW

**Comment by ckittl** [comment]
!test

File: src/test/groovy/edu/ie3/test/common/GridTestData.groovy [hunk 1]
```diff
@@ -139,7 +139,7 @@ class GridTestData {
             "node_a",
             Quantities.getQuantity(1d, PU),
             true,
-            (Point) geoJsonReader.read("{ \"type\": \"Point\", \"coordinates\": [7.411111, 51.492528] }"),
+            geoJsonReader.read("{ \"type\": \"Point\", \"coordinates\": [7.411111, 51.492528] }") as Point,
```
Comment (johanneshiry) [inline src/test/groovy/edu/ie3/test/common/GridTestData.groovy]: 👍

File: src/test/groovy/edu/ie3/datamodel/io/processor/input/GraphicInputProcessorTest.groovy [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    }
+
+    def "The GraphicInputProcessor should de-serialize a provided NodeGraphicInput with path correctly"(){
+        given:
+        GraphicInputProcessor processor = new GraphicInputProcessor(NodeGraphicInput.class)
+        NodeGraphicInput validNode = GridTestData.nodeGraphicD
+        Map expected = [
+                "uuid"          : "9ecad435-bd16-4797-a732-762c09d4af25",
+                "graphicLayer"  : "main",
+                "path"          : "{\"type\":\"LineString\",\"coordinates\":[[-1,0.0],[1,0.0]],\"crs\":{\"type\":\"name\",\"properties\":{\"name\":\"EPSG:4326\"}}}",
```
Comment (johanneshiry) [inline src/test/groovy/edu/ie3/datamodel/io/processor/input/GraphicInputProcessorTest.groovy]: Why do nodes have a path input?!

Comment (ckittl) [reply src/test/groovy/edu/ie3/datamodel/io/processor/input/GraphicInputProcessorTest.groovy]: This meant to be used, if you would like to draw the node as a bus bar (a line). The point is used, if the node is drawn as a point - what surprise 😃.

**Comment by ckittl** [comment]
!test

**Comment by johanneshiry** [comment]
test this please.

**Comment by johanneshiry** [comment]
test this please.

**Comment by johanneshiry** [comment]
!test

**Pull request merged**